src/test/java/uk/gov/hmcts/ccd/v2/external/controller/CaseControllerTestIT.java

```diff
@@ -798,4 +798,106 @@ private SupplementaryDataUpdateRequest createSupplementaryDataIncrementRequest()
         return new SupplementaryDataUpdateRequest(requestData);
     }
 
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_cases.sql"})
+    void testShouldGetLinkedCases() throws Exception {
+        final String caseId = "1504259907353529";
+        final String URL = "/getLinkedCases/" + caseId;
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+        final MvcResult mvcResult = mockMvc.perform(get(URL)
+                .header(REQUEST_ID, REQUEST_ID_VALUE)
+                .contentType(JSON_CONTENT_TYPE))
+            .andReturn();
+
+        org.assertj.core.api.Assertions.assertThat(mvcResult.getResponse())
+            .isNotNull()
+            .satisfies(response -> org.assertj.core.api.Assertions.assertThat(response.getStatus()).isEqualTo(200));
+    }
+
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_cases.sql"})
+    void testShouldGetLinkedCasesOptionalParameters() throws Exception {
+        final String caseId = "1504259907353529";
+        final String URL = "/getLinkedCases/" + caseId + "?startRecordNumber=2&?maxReturnRecordCount=1";
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+        final MvcResult mvcResult = mockMvc.perform(get(URL)
+                .header(REQUEST_ID, REQUEST_ID_VALUE)
+                .contentType(JSON_CONTENT_TYPE))
+            .andReturn();
+
+        org.assertj.core.api.Assertions.assertThat(mvcResult.getResponse())
+            .isNotNull()
+            .satisfies(response -> org.assertj.core.api.Assertions.assertThat(response.getStatus()).isEqualTo(200));
```

Comment:
Could use `HttpStatus.OK.value()`, rather than `200`, but as is its consistent with existing tests in this class, so probably leave for now

Reply:
left as is